### PR TITLE
Derive Hash and add missing import to tests

### PR DESCRIPTION
**Pull request by Person-93**
I did two things:

1. Add Hash to the list of derives
2. Add missing import to test mod. When I tried to run the tests with serde enabled, it didn't compile.

### CODE REVIEW

**Review by yihuang — APPROVED** [review]

File: src/lib.rs [hunk 1]
```diff
@@ -232,6 +232,8 @@ macro_rules! ne_vec {
 
 #[cfg(test)]
 mod tests {
+    use super::*;
```
Comment (yihuang) [inline src/lib.rs:235]: I'm not sure why, but with this line, there's `unused_imports` warning, and the test runs ok without it.

**Review by yihuang — COMMENTED** [review]
The `use` in tests is not needed?

File: src/lib.rs (hunk 1 above)
Comment (Person-93) [reply src/lib.rs:235]: When you run the tests without serde, it's unused. I generally use cargo-test-all-features in my projects to make sure everything is covered.

File: src/lib.rs [hunk 2]
```diff
@@ -232,6 +232,8 @@ macro_rules! ne_vec {
...
+
     #[cfg(feature = "serde")]
```
Comment (Crytochain) [inline src/lib.rs:237]:
```suggestion
```

Comment (Crytochain) [reply src/lib.rs:237]: Could you remove this too?

Comment (yihuang) [inline src/lib.rs]:
```suggestion
```

**Review by yihuang — APPROVED** [review]

**Pull request merged**